src/app/dashboard/page.tsx

```diff
@@ -343,11 +403,12 @@ const Home: NextPage = () => {
             </Box>
           </Flex>
         </GridItem>
+
         <GridItem colSpan={{ base: 12, lg: 12 }}>
           <Box mb={6}>
             <GraphComponent
               graphData={treasuryGraphData}
-              graphCurrentValue={getCurrentHolderValue(treasuryGraphData)}
+              graphCurrentValue={treasuryValue}
```

Comment:
I think it would make sense to refactor and create his own component instead of adding more complexity to the page. 

https://matheus.ro/2017/12/11/clean-code-boy-scout-rule/

code will be much more understandable if we can move 100 lines out of this page component into his own component 👍🏻